dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/AlertGroupMapper.java

```diff
@@ -82,4 +83,13 @@
      * @return
      */
     String queryAlertGroupInstanceIdsById(@Param("alertGroupId") int alertGroupId);
+
+    /**
+     * alert group page
+     * @param page page
+     * @param groupName groupName
+     * @return map IPage
+     */
+    IPage<Map<String, Object>> queryAlertGroup(Page<Map<String, Object>> page,
```

Comment:
It might be better to change this method name to `queryAlertGroupName` or add a description on the comments.
Someone might use this method to get the real `AlertGroup`.
And if you use map as the result, it's necessary to declare the mapping content.

Reply:
I have change this method name to `queryAlertGroupName` and add comments.